dle/search/static/search/es_search.js

```diff
@@ -31,90 +31,134 @@
             }
           ],
         }
-      })
-  
-      const search = instantsearch({
-        indexName: "productsection",
-        searchClient: SearchkitInstantsearchClient(sk)
-      });
-  
-      search.addWidgets([
-          instantsearch.widgets.searchBox({
-              queryHook(query, search) {
-                  globalSearchTerm = query;
-                  search(query);
-              },
-              container: "#searchbox"
-          }),
-          instantsearch.widgets.currentRefinements({
-              container: "#current-refinements"
-          }),
-          instantsearch.widgets.menuSelect({
-              container: "#section-name-filter",
-              attribute: "section_name",
-              field: "section_name.keyword",
-              limit: 100
-          }),
-          instantsearch.widgets.refinementList({
-              container: "#drug-label-source-filter",
-              attribute: "drug_label_source",
+    }
+)
+
+async function vectorizeText(query){
+    const response = await fetch("http://localhost:8000/api/v1/vectorize", {
```

Comment:
We removed another hardcoded localhost url in js in another file in this PR - do we wanna do the same here?

Reply:
Good catch thanks will do